pilot/pkg/serviceregistry/external/servicediscovery.go

```diff
@@ -15,24 +15,93 @@
 package external
 
 import (
```

Comment:
Can you also rename the folder please?

Reply:
Later PR - don't want to get into a naming debate :-)

I'm planning to merge this into the 'aggregate' - so the caching can be applied to all registries,
since this is another case of 'delegation' not a real registry. But after 0.8